Source/WebCore/svg/animation/SMILTime.cpp

```diff
@@ -30,36 +30,11 @@
 
 namespace WebCore {
 
-const double SMILTime::unresolvedValue = DBL_MAX;
-// Just a big value smaller than DBL_MAX. Our times are relative to 0, we don't really need the full range.
-const double SMILTime::indefiniteValue = FLT_MAX;    
-
-SMILTime operator+(const SMILTime& a, const SMILTime& b)
-{
-    if (a.isUnresolved() || b.isUnresolved())
-        return SMILTime::unresolved();
-    if (a.isIndefinite() || b.isIndefinite())
-        return SMILTime::indefinite();
-    return a.value() + b.value();
-}
-
-SMILTime operator-(const SMILTime& a, const SMILTime& b)
-{
-    if (a.isUnresolved() || b.isUnresolved())
-        return SMILTime::unresolved();
-    if (a.isIndefinite() || b.isIndefinite())
-        return SMILTime::indefinite();
-    return a.value() - b.value();
-}
-
 SMILTime operator*(const SMILTime& a,  const SMILTime& b)
 {
-    if (a.isUnresolved() || b.isUnresolved())
-        return SMILTime::unresolved();
+    // Equal operators have to be used instead of negation here to make NaN work as well.
```

Comment:
I don't see this change

Reply:
The change was `if (a.value() == 0 || b.value() == 0)` which as per WebKit style guideline would be `if (!a.value() || !b.value())`, which is below.